.gitignore

```diff
@@ -75,3 +76,18 @@ fabric.properties
 .project
 .settings/
 .classpath
+
+#IntelliJ Project files
+eclipse-collections-code-generator/*.iml
+eclipse-collections-code-generator-maven-plugin/*.iml
+eclipse-collections-api/*.iml
+eclipse-collections/*.iml
+eclipse-collections-testutils/*.iml
+eclipse-collections-forkjoin/*.iml
+unit-tests/*.iml
+serialization-tests/*.iml
+jcstress-tests/*.iml
+junit-trait-runner/*.iml
+unit-tests-java8/*.iml
+test-coverage/*.iml
+p2-repository/*.iml
```

Comment:
Replace this whole section with `*.iml`. In other words, get rid of the directory prefixes. No need for globs that only match one file.

Reply:
The global *.iml doesn’t seem to work. I have tried it before.